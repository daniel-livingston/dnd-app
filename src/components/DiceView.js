import React from "react";
import DiceForm from "./DiceForm";
import DiceHistory from "./DiceHistory";
import View from "./View";

const DiceView = () => (
	<View
		title='Dice'
		description='Customize your dice rolls and share your rolls with everyone in the room.'
	>
		<div className='dice-wrapper'>
			<div className='dice-container'>
				<DiceForm />
				<DiceHistory />
			</div>
		</div>
	</View>
);

export default DiceView;
