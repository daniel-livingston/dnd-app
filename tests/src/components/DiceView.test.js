import React from "react";
import { shallow } from "enzyme";
import DiceView from "../../../src/components/DiceView";

describe("<DiceView>", () => {
	test("renders correctly", () => {
		const wrapper = shallow(<DiceView />);
		expect(wrapper).toMatchSnapshot();
	});
});
