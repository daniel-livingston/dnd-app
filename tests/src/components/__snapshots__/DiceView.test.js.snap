// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DiceView> renders correctly 1`] = `
<Connect(View)
  description="Customize your dice rolls and share your rolls with everyone in the room."
  title="Dice"
>
  <div
    className="dice-wrapper"
  >
    <div
      className="dice-container"
    >
      <Connect(DiceForm) />
      <Connect(DiceHistory) />
    </div>
  </div>
</Connect(View)>
`;
